packages/llvm_libs.rb

```diff
@@ -0,0 +1,44 @@
+require 'package'
+require_relative 'llvm'
+
+class Llvm_libs < Package
+  description 'LLVM Libraries'
+  homepage Llvm.homepage
+  version Llvm.version
+  # When upgrading llvm, be sure to upgrade llvm_libs in tandem.
+  puts "#{self} version differs from llvm version #{Llvm.version}".orange if version != Llvm.version.to_s
```

Comment:
Will this ever print?  You're setting it on the line above.

Reply:
Ah!

Reply:
fixed.